tests/core/test_macros.py

```diff
@@ -168,6 +168,26 @@ def test_ast_correctness(macro_evaluator):
             "SELECT * FROM city",
             {"do_order": False},
         ),
+        (
+            """select @if(TRUE, 1, 0)""",
+            "SELECT 1",
+            {},
+        ),
+        (
+            """select @if(FALSE, 1, 0)""",
+            "SELECT 0",
+            {},
+        ),
+        (
+            """select @if(1 > 0, 1, 0)""",
+            "SELECT 1",
+            {},
+        ),
+        (
+            """select @if('a' = 'b', c), d""",
```

Comment:
Still okay if we had:

"""select @if('a' = 'b', c), @if('a' = 'b', foo), d"""

which would be

select ,, d

Reply:
sqlglot seems okay with it. Interesting feature that will be good for the docs.